### ISSUE TITLE
Document how to use API to get raw data

### ISSUE THREAD

**Issue by dethe** [participedia/api#213]
Eventually we'll want examples showing how to use the data, CSV support, etc. but for now just the URLs to get raw data in JSON should be documented.

**Comment by jesicarson** [comment]
@paninee should I close this, or do we need this documentation? Thanks.

**Comment by paninee** [comment]
@jesicarson What are we looking for in this issue? We have CSV support to download the data now. Looks like this issue is outdated, from 2017.

**Comment by jesicarson** [comment]
Yes it's old - just wondering if it might be useful to document how to get the raw data in JSON (in addition to the CSV urls that we have made available). If you don't know, that's ok. We can return to this later if needed.
